flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/plan/metadata/MetadataTestUtil.scala

```diff
@@ -252,9 +253,13 @@ object MetadataTestUtil {
     getMetadataTable(fieldNames, fieldTypes, new FlinkStatistic(tableStats))
   }
 
+  val tableConfig = TableConfig.getDefault
```

Comment:
I didn't inline those cases, because I'm using the `tableConfig.getConfiguration` as plannerConfig so that it's shared, in case, in a test modifies only tableConfig, it will automatically get the same change in plannerConfig.

Those will be cleaned up when we remove the tableConfig completely from the context.